src/data/resolvers/queries/knowledgebase.js

```diff
@@ -0,0 +1,120 @@
+import { KbTopics, KbCategories, KbArticles, Users } from '../../../db/models';
+
+export default {
+  kbTopic(root, { topicId }) {
```

Comment:
Do we really need to add `kb` before query names? They are inside their own module.

Reply:
If in understand correctly then by these names these back-ends get called in apollo.